azure-iot-hub-devicesdk/samples/advanced-edge-scenarios/README.md

```diff
@@ -0,0 +1,22 @@
+# Advanced IoT Edge Scenario Samples for the Azure IoT Hub Device SDK
+
+This directory contains samples showing how to use the various features of Azure IoT Hub Device SDK with Azure IoT Edge.
+
+**These samples are written to run in Python 3.7+**, but can be made to work with Python 3.5 and 3.6 with a slight modification as noted in each sample:
+
+```python
+if __name__ == "__main__":
+    asyncio.run(main())
+
+    # If using Python 3.6 or below, use the following code instead of asyncio.run(main()):
+    # loop = asyncio.get_event_loop()
+    # loop.run_until_complete(main())
+    # loop.close()
+```
+
+In order to use these samples, they **must** be run from inside an Edge container.
```

Comment:
We need more detail here, and really, in this README at large. The Edge scenario story isn't well defined, but since we aren't really showing off this functionality right now, I figured a simple skeleton would be sufficient.